app/models/subnet.rb

```diff
@@ -213,6 +213,12 @@ def has_vlanid?
     self.vlanid.present?
   end
 
+  # Given a parameter name, return the (first) value, or nil if nothing found
+  def parameter_for_name(name)
```

Comment:
Since we have `host_param`, I suppose it'd make sense to have a similar syntax for fetching Subnet parameters, so `subnet_param` - checkout https://github.com/theforeman/foreman/blob/develop/lib/foreman/renderer.rb#L36 or https://github.com/theforeman/foreman/commit/477b6b6b932ee27b2b1c6b2ff39ae6b07fe120e9 if you want to see how it was implemented

@ares @xprazak2 Is it possible to log something here when the parameter is not found, but make it show up on the warning dialog during preview instead of the foreman logs?